### PR TITLE
Include email reporters on Documentation

### PR DESCRIPTION
**Pull request by verracca**

### CODE REVIEW

File: modules/ROOT/pages/bat-reporting-task.adoc [hunk 1]
```diff
@@ -233,4 +234,20 @@ And the result is:
 {
   "result": "The suite <Hello world suite> with assertion <answer 200> is ok"
 }
-----
\ No newline at end of file
+----
+
+== Email
+
+When using the Anypoint Platform you can create an email reporter to enable email alerts to several accounts.
```
Comment (rddheath) [inline modules/ROOT/pages/bat-reporting-task.adoc]: What do you mean by "when using the Anypoint Platform"? Using it in what way? 
Do the listed email addresses need to be associated with accounts in Anypoint Platform?

File: modules/ROOT/pages/bat-reporting-task.adoc [hunk 2]
```diff
@@ -233,4 +234,20 @@ And the result is:
...
+
+_This option is only available in the Anypoint Platform since it uses the Anypoint Platform Alert Notification System._
```
Comment (rddheath) [inline modules/ROOT/pages/bat-reporting-task.adoc]: How does this system need to be configured?

Comment (ferferrarazzo) [reply modules/ROOT/pages/bat-reporting-task.adoc]: @rddheath I changed the doc. can you take a look?

**Comment by rddheath** [comment]
I'll have to review this next week, when I can spend more time on it and ask you questions. This week is packed.

**Review by rddheath — APPROVED** [review]
Do not merge this PR, yet. If you do, the changes will go to the Docs team's staging server and will be published today. We need to wait until Friday to merge the PR.

**Pull request merged**